src/main/java/net/dv8tion/jda/api/events/thread/member/GenericThreadMemberEvent.java

```diff
@@ -25,7 +25,15 @@
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 
-//TODO-v5: Docs
+/**
+ * Indicates that a {@link Member GuildMember} interacted with a {@link ThreadChannel} as a {@link ThreadMember}.
```

Comment:
Interacted seems like a strange wording here, can't a member also be added by someone else (private threads) without "interacting"? Especially problematic because "interaction" is a defined term for a specific API.

Reply:
I rephrased it similar to the GenericGuildMember event